src/managed/OpenLiveWriter.PostEditor/Video/VideoProvidersB2.xml

```diff
@@ -77,30 +77,20 @@
   <provider>
     <id>56744E0A-1892-4b56-B7FF-3A2568EE2D27</id>
     <serviceName>YouTube</serviceName>
-    <editorFormat xpp:if="version lt 15.4.3104.0816">
-      <![CDATA[
-		<embed src="http://www.youtube.com/v/{videoId}" type="application/x-shockwave-flash" wmode="transparent" width="{width}" height="{height}"></embed>
-		]]>
-    </editorFormat>
-    <editorFormat xpp:if="version gte 15.4.3104.0816">
+    <editorFormat>
       <![CDATA[
-		<embed src="http://www.youtube.com/v/{videoId}?hd=1" type="application/x-shockwave-flash" wmode="transparent" width="{width}" height="{height}"></embed>
+		<iframe src="http://www.youtube.com/embed/{videoId}" width="{width}" height="{height}" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 		]]>
     </editorFormat>
-    <embedFormat xpp:if="version lt 15.4.3104.0816">
-      <![CDATA[
-      <object width="{width}" height="{height}"><param name="movie" value="http://www.youtube.com/v/{videoId}&hl=en"></param><embed src="http://www.youtube.com/v/{videoId}&hl=en" type="application/x-shockwave-flash" width="{width}" height="{height}"></embed></object>
-		]]>
-    </embedFormat>
-    <embedFormat xpp:if="version gte 15.4.3104.0816">
+    <embedFormat>
       <![CDATA[
-      <object width="{width}" height="{height}"><param name="movie" value="http://www.youtube.com/v/{videoId}?hl=en&hd=1"></param><embed src="http://www.youtube.com/v/{videoId}?hl=en&hd=1" type="application/x-shockwave-flash" width="{width}" height="{height}"></embed></object>
+      <object width="{width}" height="{height}"><param name="movie" value="http://www.youtube.com/embed/{videoId}&hl=en"></param><iframe src="http://www.youtube.com/embed/{videoId}&hl=en" width="{width}" height="{height}" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></object>
```

Comment:
It seems wrapping the `iframe` in the `object` tag breaks videos on Blogger. I think we should be able to remove the tag and just have the `iframe` alone 👍

Reply:
Sure, I have removed the ```object``` tag, updated the changes and checked it too. 👍

Reply:
Sorry for not being clear enough I checked that in Wordpress only and there it was working. Also, I was not able to login to blogger while debugging. It was getting 401 error. I think first we have to login through the default installed the application. Or Am I doing it wrong 🤔